optuna/integration/chainer.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import absolute_import
 
 import optuna
+import optuna.exceptions
```

Comment:
I think this import is redundant and can be removed.

Ditto:
- https://github.com/pfnet/optuna/pull/691/files#diff-8141f4583ec32199544b27faaae78294R4
- https://github.com/pfnet/optuna/pull/691/files#diff-d2a4d1ffb63cfa539d5f3ade6ea0f904R4
- https://github.com/pfnet/optuna/pull/691/files#diff-456cd17454181c4f30511dc3c6d058c2R5
- https://github.com/pfnet/optuna/pull/691/files#diff-2f109b7ae58da8b64fceb79b5d6f1772R4
- https://github.com/pfnet/optuna/pull/691/files#diff-57eed777b6869d898e56a05b83153f1dR2
- https://github.com/pfnet/optuna/pull/691/files#diff-37446c9777fc86723ebc01eb61ccb696R4
- https://github.com/pfnet/optuna/pull/691/files#diff-6777d6b27690da66a18f118b10e4a1dcR4
- https://github.com/pfnet/optuna/pull/691/files#diff-3929ab794590d8de23a3b0c4c942a1edR4
- https://github.com/pfnet/optuna/pull/691/files#diff-7fabbd1da1bd3bebde0e04965f81c892R4
- https://github.com/pfnet/optuna/pull/691/files#diff-70f8b9dc0cc3d59bab2c32029cf14808R23
- https://github.com/pfnet/optuna/pull/691/files#diff-19ce543d124092cc29a1e3eaeeb8617bR7
- https://github.com/pfnet/optuna/pull/691/files#diff-b106f266d7dc791aa59a717e323c11fcR6
- https://github.com/pfnet/optuna/pull/691/files#diff-067702c8553ea58a6a902282be88e745R7
- https://github.com/pfnet/optuna/pull/691/files#diff-cff9f9d21d151e4c36bfe2771ab85f22R7
- https://github.com/pfnet/optuna/pull/691/files#diff-c17213c19ecafb8f54ec76a10011264cR5
- https://github.com/pfnet/optuna/pull/691/files#diff-48005dfcd962e57ca1be2ec86e03144eR14

Reply:
You're right. I removed them in the commit https://github.com/pfnet/optuna/pull/691/commits/1aaae9676782a37170824a848f3f2b7f6877b941.
Thank you!